### PR TITLE
Fix math equation rendering format in nn related doc strings

### PR DESCRIPTION
**Pull request by imsheridan**
This PR is to fix mess-up math equation format in below NN related docstrings.

This PR is to fix this math equation issue according to the [Math in markdown guideline](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/community/documentation.md#math-in-markdown).

Take [sparse_softmax_cross_entropy_with_logits](https://www.tensorflow.org/api_docs/python/tf/nn/sparse_softmax_cross_entropy_with_logits) as an example below:
Before:
image
After:
image

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrewharp: It has been 53 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrewharp: It has been 68 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrewharp: It has been 83 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrewharp: It has been 98 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrewharp: It has been 113 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrewharp: It has been 127 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

File: tensorflow/python/ops/nn_ops.py [hunk 1]
```diff
@@ -682,16 +682,14 @@ def convolution(
   of N `strides` (defaulting [1]*N), this computes for each N-D spatial output
   position (x[0], ..., x[N-1]):
 
-  ```
     output[b, x[0], ..., x[N-1], k] =
-        sum_{z[0], ..., z[N-1], q}
+        \\(sum_{z[0], ..., z[N-1], q}\\)
```
Comment (annarev) [inline tensorflow/python/ops/nn_ops.py]: These equations look longer but your change only converts 1 line. Should entire equations be surrounded by \\( \\)? How would it look if only 1 line is converted?

**Comment by tensorflowbutler** [comment]
It has been 20 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 49 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
